src/mh-core.jl

```diff
@@ -196,15 +196,15 @@ function AbstractMCMC.step!(
     model::DensityModel,
     spl::MetropolisHastings,
     ::Integer,
-    params_prev::Transition;
+    params_prev;
     kwargs...
 )
     # Generate a new proposal.
     params = propose(rng, spl, model, params_prev)
 
     # Calculate the log acceptance probability.
-    logα = logdensity(model, params) - logdensity(model, params_prev) + 
-        q(spl, params_prev, params) - q(spl, params, params_prev)
+    logα = logdensity(model, params) - logdensity(model, params_prev) 
+        + q(spl, params_prev, params) - q(spl, params, params_prev)
```

Comment:
Are you sure this is still doing the right thing? I would assume it just sets logalpha to `logdensity(...) - logdensity(...)`, without considering the line below.
```suggestion
    logα = logdensity(model, params) - logdensity(model, params_prev) +
        q(spl, params_prev, params) - q(spl, params, params_prev)
```

Reply:
I don't think it is doing what I thought, but I'm also now not sure if I need the second part at all (the `q(. . .)` calls).

I think it's just supposed to be the different in logjoints -- would you agree? ([source](https://projecteuclid.org/download/pdf_1/euclid.camcos/1513731992))

Reply:
Oh, wait, my bad. I thought I was looking at the ensemble stuff and not the MH stuff. Ignore me.